cmd/dispatcher/main.go

```diff
@@ -103,13 +103,14 @@ func main() {
 	rmqHelper := rabbit.NewRabbitMQHelper(1, make(chan bool), rabbit.DialWrapper)
 	defer rmqHelper.CleanupRabbitMQ(env.connection, logger)
 	for {
-		env.connection, env.channel, err = rmqHelper.SetupRabbitMQ(env.RabbitURL, rabbit.ChannelQoS, logger)
+		env.connection, env.channel, err = rmqHelper.SetupRabbitMQ(env.RabbitURL, rabbit.ChannelQoS, logger, true)
 		if err != nil {
 			logger.Errorf("error creating RabbitMQ connections: %s, waiting for a retry", err)
 		} else if err := d.ConsumeFromQueue(ctx, env.channel, env.QueueName); err != nil {
-			if errors.Is(err, context.Canceled) {
-				return
+			if !errors.Is(err, context.Canceled) {
+				logger.Errorf("error consuming from RabbitMQ: %s", err)
```

Comment:
In the dispatcher's case, if there is an error with `ConsumeFromQueue` that isn't connection related, do we just wait for the retry signal on 116 indefinitely?

I know this bug isn't really to fix dispatcher but we'd probably want some consistent behaviours